magefile.go

```diff
@@ -984,6 +984,35 @@ func (Cloud) Image(ctx context.Context) {
 	Package(ctx)
 }
 
+// Load loads an artifact as a docker image.
+// Looks in build/distributions for an elastic-agent-cloud*.docker.tar.gz artifact and imports it as docker.elastic.co/beats-ci/elastic-agent-cloud:$VERSION
+// DOCKER_IMPORT_SOURCE - override source for import
+func (Cloud) Load() error {
```

Comment:
Why do we need this implemented as a mage target?
If I understand the flow correctly:
1. image is built and saved in the packaging step
2. image is downloaded in buildkite `Build and push custom elastic-agent image` (spoiler alert: we are not building anything here!)
3.  `mage cloud:load` and `mage cloud:push` are called but they simply perform a `docker load` and a `docker push` from shell
4. control go back to buildkite that passes the pushed image tag (which is present as an env var) to terraform to create the ECH stack.

Why is this back and forth needed between buildkite and mage? Wouldn't this be simpler if the docker load and docker push was called directly from buildkite ?

Reply:
I added `load` to be consistent with our other mage targets
Why do we have `cloud:push` as a mage target?